packages/smart-contracts/src/contracts/ERC20TransferrableReceivable.sol

```diff
@@ -0,0 +1,170 @@
+// Contract based on https://docs.openzeppelin.com/contracts/3.x/erc721
```

Comment:
This line is too long [according to the linter](https://app.circleci.com/pipelines/github/RequestNetwork/requestNetwork/7804/workflows/246b385c-1ec2-4a4f-9396-98152fe9a913/jobs/127314?invite=true#step-103-8). Can you remove the word "Contract" or shorten it some other way?
```suggestion
// Based on [https://docs.openzeppelin.com/contracts/3.x/erc721](https://docs.openzeppelin.com/contracts/3.x/erc721)
```